api/checks/update_test.go

```diff
@@ -0,0 +1,76 @@
+// +build small
+
+package checks
+
+import (
+	"context"
+	"net/url"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/web-platform-tests/wpt.fyi/api/checks/summaries"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+	"github.com/web-platform-tests/wpt.fyi/shared/sharedtest"
+)
+
+func TestGetDiffSummary_Regressed(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	before := shared.TestRun{}
+	before.FullRevisionHash = strings.Repeat("0", 40)
+	before.BrowserName = "chrome"
+	after := shared.TestRun{}
+	after.FullRevisionHash = strings.Repeat("1", 40)
+	after.BrowserName = "chrome"
+	runDiff := shared.RunDiff{
+		Differences: shared.ResultsDiff{"/foo.html": shared.TestDiff{0, 1, 0}},
```

Comment:
nit: probably can't hurt to comment why a TestDiff of 0,1,0 is a regression while 1,0,1 is a completion

Reply:
Done via a little overkill assertion.